projects/netgrif-components/src/lib/data-fields/number-field/number-currency-field/number-currency-field.component.html

```diff
@@ -8,7 +8,9 @@
            [placeholder]="numberField.placeholder"
            [required]="numberField.behavior.required"
            (focusout)="onFocusOut($event)"
-           (focusin)="onFocusIn()">
+           (focusin)="onFocusIn()"
```

Comment:
working correctly ? because there are assigned two different functions two times, isn't it better to assigned it like (focusin)="onFocusIn();numberField.setFocus()" ?